beekeeper-studio.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -oue pipefail
+
+export FLATPAK_ID="${FLATPAK_ID:-io.beekeeperstudio.Studio}"
+export TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
+
+# Wayland support can be optionally enabled like so:
+# flatpak override --user --env=USE_WAYLAND=1 io.beekeeperstudio.Studio
+declare -i USE_WAYLAND="${USE_WAYLAND:-0}"
```

Comment:
not with the socket permissions.

if upstream doesn't support wayland out of the box, flatpak definitely shouldn't

Reply:
If Ozone and Wayland are not enabled by upstream by default, I don't think it makes much sense to add it in the script.